api/src/main/java/io/smallrye/stork/api/Metadata.java

```diff
@@ -119,4 +118,12 @@ public String getName() {
         }
     }
 
+    public enum NoMetadataKey implements MetadataKey {
+        ;
```

Comment:
why the semi-colon?

Reply:
Isn't it needed between constants/values declaration and methods?

Reply:
yes, it's needed, otherwise it doesn't compile...